config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "public_recipes#index"
+
+  # get "/public_recipes", to: 'public_recipes#index'
+  # get "/public_recipes/:id", to: 'public_recipes#show'
```

Comment:
- [ Required ] I think it would be better if you remove commented code from your code base, commented code makes your repository look unprofessional. Please rectify this.